src/steps/smart-campaign-add-lead.ts

```diff
@@ -18,6 +18,29 @@ export class AddLeadToSmartCampaignStep extends BaseStep implements StepInterfac
     type: FieldDefinition.Type.ANYSCALAR,
     description: 'Smart campaign name or numeric id',
   }];
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'lead',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
+      field: 'id',
+      type: FieldDefinition.Type.NUMERIC,
+      description: "Lead's Marketo ID",
+    }, {
+      field: 'email',
+      type: FieldDefinition.Type.EMAIL,
+      description: "Lead's Email",
+    }],
+    dynamicFields: true,
+  }, {
+    id: 'campaign',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
```

Comment:
- [ ] Would recommend adding `createdAt`, `updatedAt`, and `name` to expected fields, here.